### PR TITLE
ADDS ALL CONTRIBUTORS 

### PR DESCRIPTION
**Pull request by pnkjxmwl**
image

### CODE REVIEW

**Comment by pnkjxmwl** [comment]
if you accept my pull request please add a label with hacktoberfest-accepted and merge it. @ShehrozIrfan

File: readme.md [hunk 1]
```diff
@@ -8,3 +8,22 @@
 
 ## What is Hacktoberfest?
 Hacktoberfest is a month-long celebration of open source software run by DigitalOcean in partnership with GitHub and Twilio. Hacktoberfest is open to everyone in our global community! Four quality pull requests must be submitted to public GitHub repositories. You can sign up anytime between October 1 and October 31.
+
+## Contributors ✨
+
+Thanks goes to these wonderful people 💜
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/ShehrozIrfan"><img src="https://avatars.githubusercontent.com/u/36946330?s=60&v=4" width="100px;" alt=""/><br /><sub><b>ShehrozIrfan</b></sub></a><br /><a href="#maintenance-Tlazypanda" title="Maintenance">🚧</a></td>
```
Comment (ShehrozIrfan) [inline readme.md]: It's the great work.
But every time there is a new contributor we've to add it manually.
Is there any way it is added automatically?

File: readme.md [hunk 2]
```diff
@@ -8,3 +8,22 @@
 
 ## What is Hacktoberfest?
 Hacktoberfest is a month-long celebration of open source software run by DigitalOcean in partnership with GitHub and Twilio. Hacktoberfest is open to everyone in our global community! Four quality pull requests must be submitted to public GitHub repositories. You can sign up anytime between October 1 and October 31.
+
+## Contributors ✨
+
+Thanks goes to these wonderful people 💜
```
Comment (ShehrozIrfan) [inline readme.md]: Change the text to:
`Our Valuable Contributors`

File: readme.md [hunk 3]
```diff
@@ -8,3 +8,22 @@
...
+    <td align="center"><a href="https://github.com/AnshumanGoel"><img src="https://avatars.githubusercontent.com/u/99547341?s=60&v=4" width="100px;" alt=""/><br /><sub><b>Anshuman Goel</b></sub></a><br /> <a  title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/Arika008"><img src="https://avatars.githubusercontent.com/u/100206199?s=60&v=4" width="100px;" alt=""/><br /><sub><b>Arika Pandey</b></sub></a><br /><a  title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/SiddharthBihola"><img src="https://avatars.githubusercontent.com/u/112693399?s=60&v=4" width="100px;" alt=""/><br /><sub><b>SiddharthBihola</b></sub></a><br /><a title="Code">💻</a></td>
+    </tr>
+    <tr>
+    <td align="center"><a href="https://github.com/Sayan2908"><img src="https://avatars.githubusercontent.com/u/107147382?s=60&v=4" width="100px;" alt=""/><br /><sub><b>Sayan Dey Sarkar</b></sub></a><br /><a title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/yash-sharma13"><img src="https://avatars.githubusercontent.com/u/95098944?s=60&v=4" width="100px;" alt=""/><br /><sub><b>yash sharma</b></sub></a><br /><a title="Code">💻</a></td>
+  </tr>
+
```
Comment (ShehrozIrfan) [inline readme.md]: remove this extra line.

**Comment by pnkjxmwl** [comment]
Hey Now i Have Automated the Contributors part .

**Comment by ShehrozIrfan** [comment]
@pnkjxmwl 
Excellent.
You can follow me and star this repository for further updates.
Have fun 🙌🏼

**Pull request merged**